2018-06-04 21:56:24 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-04 21:58:05 No exception in execute_script method.
2018-06-04 22:10:14 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-04 22:10:35 No exception in execute_script method.
2018-06-04 22:14:23 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-04 22:14:44 No exception in execute_script method.
2018-06-04 22:14:44 execute_script :I:\PythonUnittest\scripts\test2.py
2018-06-04 22:14:44 No exception in execute_script method.
2018-06-04 22:14:44 execute_script :I:\PythonUnittest\scripts\test3.py
2018-06-04 22:14:44 No exception in execute_script method.
2018-06-04 22:32:10 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-04 22:32:31 No exception in execute_script method.
2018-06-04 22:37:39 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-04 22:38:00 No exception in execute_script method.
2018-06-04 22:40:33 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-04 22:40:55 No exception in execute_script method.
2018-06-04 22:40:55 execute_script :I:\PythonUnittest\scripts\test2.py
2018-06-04 22:40:55 No exception in execute_script method.
2018-06-04 22:40:55 execute_script :I:\PythonUnittest\scripts\test3.py
2018-06-04 22:40:55 No exception in execute_script method.
2018-06-04 23:14:16 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-04 23:14:27 No exception in execute_scriptmethod
2018-06-04 23:14:27 execute_script :I:\PythonUnittest\scripts\test2.py
2018-06-04 23:14:27 No exception in execute_scriptmethod
2018-06-04 23:14:27 execute_script :I:\PythonUnittest\scripts\test3.py
2018-06-04 23:14:27 No exception in execute_scriptmethod
2018-06-04 23:41:32 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-04 23:41:43 No exception in execute_scriptmethod
2018-06-04 23:49:31 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-04 23:49:42 No exception in execute_scriptmethod
2018-06-04 23:59:00 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-04 23:59:11 No exception in execute_script  method
2018-06-04 23:59:11 execute_script :I:\PythonUnittest\scripts\test2.py
2018-06-04 23:59:11 No exception in execute_script  method
2018-06-04 23:59:11 execute_script :I:\PythonUnittest\scripts\test3.py
2018-06-04 23:59:11 No exception in execute_script  method
2018-06-05 00:02:17 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-05 00:02:28 No exception in execute_script  method
2018-06-05 00:02:28 execute_script :I:\PythonUnittest\scripts\test2.py
2018-06-05 00:02:28 No exception in execute_script  method
2018-06-05 00:02:28 execute_script :I:\PythonUnittest\scripts\test3.py
2018-06-05 00:02:28 No exception in execute_script  method
2018-06-05 00:19:13 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-05 00:19:24 No exception in execute_script  method
2018-06-05 00:19:24 execute_script :I:\PythonUnittest\scripts\test2.py
2018-06-05 00:19:24 No exception in execute_script  method
2018-06-05 00:19:24 execute_script :I:\PythonUnittest\scripts\test3.py
2018-06-05 00:19:25 No exception in execute_script  method
2018-06-05 00:39:02 execute_script :I:\PythonUnittest\scripts\test.py
2018-06-05 00:39:13 No exception in execute_script  method
2018-06-05 00:39:13 execute_script :I:\PythonUnittest\scripts\test2.py
2018-06-05 00:39:13 No exception in execute_script  method
2018-06-05 00:39:13 execute_script :I:\PythonUnittest\scripts\test3.py
2018-06-05 00:39:13 No exception in execute_script  method
